risc0/zkvm/src/host/client/prove/mod.rs

```diff
@@ -163,6 +167,8 @@ pub struct ProverOpts {
     /// programs that are allowed to run and is a key field in the
     /// [SuccinctReceiptVerifierParameters][crate::SuccinctReceiptVerifierParameters].
     pub control_ids: Vec<Digest>,
+    /// Maximum cycle count, as a power of two (po2) that these prover options support.
+    pub(crate) segment_po2_max: usize,
```

Comment:
```suggestion
    pub(crate) max_segment_po2: usize,
```

or just `max_po2` so that it aligns with `from_max_po2` and `DEFAULT_MAX_PO2`, etc.

Reply:
Addressed in b19f5e2779b3930b2512f393c87aef0cd4436991